### PR TITLE
Double_GEAR_ratio_Z

### PR DESCRIPTION
**Pull request by zzp8202**
### Which mods are added by this PR?
One 

### How was it tested?
Test by myself.

### Any background context you want to provide?
Enabling the Voron 2.4 Gantry to Defy Gravity
### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [x] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [x] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

File: printer_mods/ZZP8202/Double_GEAR_ratio_Z/Images/Photo.jpg
Comment (chirpy2605) [inline printer_mods/ZZP8202/Double_GEAR_ratio_Z/Images/Photo.jpg:1]: Could you resize this pic as it's over 2MB in size. For example, I've attached a resized one that is 10% of the file size
Photo

File: printer_mods/ZZP8202/Double_GEAR_ratio_Z/README.md
Comment (chirpy2605) [inline printer_mods/ZZP8202/Double_GEAR_ratio_Z/README.md:1]: Could you please add a section that explains exactly how to assemble this Modification on the printer using text and pictures/images to details how it goes together.

**Comment by zzp8202** [comment]
I have resize the photo , and  add assemble instructions with gif image.

**Comment by zzp8202** [comment]
I have resize the photo ,  add assemble instructions with gif image, and edit the '.metadata.yml' file.

File: printer_mods/ZZP8202/Double_GEAR_ratio_Z/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+In my modification, I used GE5C ball joints to connect the gantry and the linear rails, which should provide a more stable structure. I also used a pulley system to increase the gear ratio to 8:1, allowing the gantry to maintain its position without external forces after a power outage. Practical tests show that only one leveling attempt is needed after a power outage without external influences, compared to the previous 3-4 attempts needed to complete the leveling.
+
+
+## Image:
+![front](Images/CAD_front_view.png)
+![side](Images/CAD_side_view.png)
+![Detal2](Images/CAD_Detal_view2.png)
+![Detal1](Images/CAD_Detal_view1.png)
+![Photo](Images/ezgif.com-resize.gif)
```
Comment (chirpy2605) [inline printer_mods/ZZP8202/Double_GEAR_ratio_Z/README.md:24]: The README.md is referencing "Images/ezgif.com-resize.gif" but that file doesn't exist

File: printer_mods/ZZP8202/Double_GEAR_ratio_Z/.metadata.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- STL/Z_Bearing_Block_Top_X4.STL
+- STL/Z_Belt_Clamp_Bottom_x8.STL
+- STL/Z_Belt_Clamp_Upper_x8.STL
+images:
+- Images/CAD_Detal_view1.png
+- Images/CAD_Detal_view2.png
+- Images/CAD_front_view.png
+- Images/CAD_side_view.png
+- Images/Photo.jpg
+-Other/Assemble_Instructions/Pulley_Holder.gif
```
Comment (chirpy2605) [inline printer_mods/ZZP8202/Double_GEAR_ratio_Z/.metadata.yml:24]: There are some formatting errors in the YAML file. "The Other:" section should be first and then the subsequent entries need a space after the "-" in "-/Other/..."

Like so:
```
Other:
- Other/Assemble_Instructions/Pulley_Holder.gif
- Other/Assemble_Instructions/Z_Bearing_Block_Bottom_With_Pulley.gif
- Other/Assemble_Instructions/BOM/Pulley_Holder_X4_BOM.png
- Other/Assemble_Instructions/BOM/Z_Bearing_Block_Bottom_With_Pulley_BOM.png
- Other/Assemble_Instructions/BOM/Pulley_Holder_X4_BOM.pdf
- Other/Assemble_Instructions/BOM/Z_Bearing_Block_Bottom_With_Pulley_BOM.pdf

```

**Review by chirpy2605 — CHANGES_REQUESTED** [review]

**Comment by zzp8202** [comment]
sorry for mistake, I have fix it as your marked.

**Review by chirpy2605 — APPROVED** [review]
All looks good to me

**Comment by jontek2** [comment]
Hey @zzp8202,

Thanks for submitting, very cool mod!

Upon my review I've noticed that:

1. "/STL/Z_Bearing_Block_Bottom_With_Pulley_Left_A_x2.STL" has the Voron-logo upside down.

2. The description in the README.md is "Double_GEAR_ratio_Z_belt_assembly_with_spherical_bearing_connection". Maybe you could write a short sentence about how it can help with gantry sagging?

Other than that it looks good to me and once the two issues mentioned above has been dealt with i'll happily approve this mod.

Thanks!

**Review by jontek2 — APPROVED** [review]
LGTM!

**Pull request merged**